xmtp_mls/src/groups/device_sync/backup.rs

```diff
@@ -0,0 +1,180 @@
+use super::DeviceSyncError;
+use crate::storage::xmtp_openmls_provider::XmtpOpenMlsProvider;
+use backup_exporter::BackupExporter;
+use std::{path::Path, sync::Arc};
+use thiserror::Error;
+use xmtp_common::time::now_ns;
+use xmtp_proto::xmtp::device_sync::{BackupElementSelection, BackupMetadata};
+
+pub use backup_importer::BackupImporter;
+
+// Increment on breaking changes
+const BACKUP_VERSION: u32 = 0;
```

Comment:
what qualifies as a breaking change where we need to increment this?

Reply:
If we add a new optional field to one of the structs that's exported, we'd want to put some logic on the importer that says something like... `If backup_version < 1, set new field to this default value` or maybe use this separate hydrater entirely. That's just one example. Really this is just a field that I'm sure will be useful to delineate between updates, and make importing old archives possible.